target/classes/META-INF/resources/style.css

```diff
@@ -0,0 +1,9 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#greeting-container {
+  margin-top: 20px;
+}
```

Comment:
Neither of these rules seem to be used anywhere. Is there a reason you added them?